print-battleground-state-changes

```diff
@@ -213,14 +216,28 @@ def html_write_state_head(state: str, state_slug: str, summary: IterationSummary
     '''
 
 def html_summary(state_slug: str, summary: IterationSummary):
+    if summary.counties_partition:
+        counties_partition = sorted(summary.counties_partition.items(), key=lambda x: x[1], reverse=True)
+        counties_total = sum(summary.counties_partition.values())
+        counties_tooltip_attributes = (
+            'class="has-tip" data-toggle="tooltip" data-html="true" data-title="' +
```

Comment:
I don't think you need these +s between lines. Python will concat at interpretation time when a string starts & stops on a line, e.g.

```python
>>> a = (
... "aaa"
... "bbb"
... "ccc"
... )
>>> print(a)
aaabbbccc
```